src/common/messages.ts

```diff
@@ -155,6 +155,8 @@ export const Messages = {
         LoadAssessment: `${messagePrefix}/mediumPass/loadAssessment`,
         LoadAssessmentFinishedUpload: `${messagePrefix}/mediumPass/loadAssessmentFinishedUpload`,
         SaveAssessment: `${messagePrefix}/mediumPass/saveAssessment`,
+        InitiateTransferDataToAssessment: `${messagePrefix}/mediumPass/initiateTransferDataToAssessment`,
```

Comment:
If we're aiming to keep these actions somewhat agnostic for now, it might make sense to create a new message type for them since technically this action will touch both QuickAssess and Assessment.

Reply:
Good point! I think I'll add that in a future PR.